EIPS/eip-draft-extended-mnemonics.md

```diff
@@ -0,0 +1,143 @@
+## Preamble
+
+    EIP: <to be assigned>
+    Title: Extended mnemonic codes
+    Author: Nick Johnson <nick@ethereum.org>, Micah Zoltu
+    Type: Standard Track
+    Category: ERC
+    Status: Draft
+    Created: 2017-04-13
+    Requires: eip-draft-hdwallets
+
+
+## Abstract
+
+This EIP specifies a method for generating mnemonic codes based on [BIP39](https://github.com/bitcoin/bips/blob/master/bip-0039.mediawiki). Mnemonic codes provide an easy to remember or transcribe sequence of words that act as a seed for a deterministic wallets.
+
+This EIP extends BIP39 by providing additional metadata for applications. This metadata makes new mnemonic-based applications possible, while also helping to standardise derivation paths in Ethereum-based applications.
+
+## Motivation
+
+BIP39 defines a method for generating mnemonic codes that are easily written down and entered, and for deriving HD wallet seeds from these mnemonic codes. This allows for easy backup and recovery of cryptographic keys, eliminating user error when recording and entering them.
+
+Several Ethereum wallets have adopted BIP39 mnemonics for use on the Ethereum platform, as have some other products such as paper wallets. As a rule, adopting BIP39 without changes works well, but there are two shortcomings:
+
+ - Confusion over derivation paths for the resulting HD wallets, as described in eip-draft-hdwallets.
```

Comment:
Can you add a link to `eip-draft-hdwallets` here? It's this: https://github.com/ethereum/EIPs/pull/601 ?